dags/atd_finance_data_master_agreements.py

```diff
@@ -140,4 +140,15 @@
         mount_tmp_dir=False,
     )
 
-    t1
+    t2 = DockerOperator(
+        task_id="master_agreements_to_finance_purchasing",
+        image="atddocker/atd-finance-data:production",
+        auto_remove=True,
+        command="python s3_to_knack.py master_agreements finance-purchasing",
+        environment=finance_purchasing_env,
+        tty=True,
+        force_pull=False,
```

Comment:
i guess you don't need this on the subsequent tasks. i keep forgetting to remove it.